templates/config.json

```diff
@@ -637,6 +695,11 @@
       "provider": "",
       "summary": "Management and each individual department/team holds regular company-wide / departmental / team meetings to review and discuss various aspects of business performance and objectives.",
       "guidance": "It is important to keep everyone on team / in the company regularly updated on the overall business strategy, goals and performance. One example is weekly/monthly roundtable or townhalls. At the individual team level, this could be the daily development standups/scrum sessions. This helps fosture an engaging, transparent and fast-moving culture.",
+      "instructionsSummary": "Confirm/tailor company-wide roundtable frequency.",
+      "instructions": [
+        "What tool do you use for HIPPAA Awareness Training?",
+        "What interval do employees to have to complete training?"
```

Comment:
I don't this is called out in the procedure

Reply:
![image](https://user-images.githubusercontent.com/76698036/107298791-a80d7100-6a3b-11eb-8b82-ed8556118776.png)

Reply:
The screenshot is for `cp-training-hipaa` but this section in reference is for `cp-internal-comms.md`